denonavr/audyssey.py

```diff
@@ -94,6 +97,10 @@ async def _async_sound_detail_callback(
             self._dynamiceq = "1"
         elif parameter == "DYNEQ OFF":
             self._dynamiceq = "0"
+        elif parameter == "PSLFC ON":
+            self._lfc = "1"
+        elif parameter == "PSLFC OFF":
```

Comment:
```suggestion
        elif parameter == "LFC OFF":
```

Reply:
Right, I learned this after writing this code in my settings state pr :)